packages/foreman/foreman/foreman.spec

```diff
@@ -485,75 +484,75 @@ Requires: %{name} = %{version}-%{release}
 Requires: %{scl}-runtime-assets >= 5
 Requires: %{scl}-runtime-assets < 6
 %endif
-Requires: nodejs >= 6.10
+Requires: %{?scl_prefix_nodejs}nodejs >= 6.10
 # Temporary dep on libuv until https://bugs.centos.org/view.php?id=10606
 # is resolved
 Requires: libuv
```

Comment:
this probably can be dropped too?

Reply:
Probably... I can do that here or follow up with some clean up once we know things are fully working?

Reply:
given we dropped it from `BuildRequires` in this PR, it would be consistent to also drop it from the `Requires` (or not touch at all).